README.md

```diff
@@ -11,170 +11,62 @@ The original customized Auth0 lock, which can be found in [`mozilla-iam/auth0-de
-script which, using the [`mozilla-iam/auth0-ci`](https://github.com/mozilla-iam/auth0-ci/)
-python module installed in the [`mozillaiam/auth0-custom-lock-builder`](https://hub.docker.com/r/mozillaiam/auth0-custom-lock-builder/)
-environment image, calls the [`uploader_login_page.py` tool](https://github.com/mozilla-iam/auth0-ci/blob/master/uploader_login_page.py).
-
-`uploader_login_page.py` calls the Auth0 API to update the client attributes of a special
-"Default Client" in Auth0. This special reserved client (a relying party) has a
-special attribute called `custom_login_page` which, when updated, updates the
-hosted login page which is sourced from [`src/html/index.html`](src/html/index.html).
-
-This HTML page references the S3 CDN URL now containing the built javascript artifacts.
-
-#### Changing the AWS CodeBuild to GitHub integration
-
-These AWS CodeBuild webhooks must be provisioned by AWS CodeBuild (as opposed to
-manually within GitHub). As a result the process to create or update these
-webhooks must follow this process
-
-* As a GitHub user with admin rights in this repo, [add a new collaborator](https://github.com/mozilla-iam/auth0-custom-lock/settings/collaboration)
-  to the repo
-* The collaborator to add is
-  * the `infosec-prod-371522382791-codebuild` GitHub user to affect the `production` webhook
-  * the `infosec-dev-656532927350-codebuild` GitHub user to affect the `master` webhook
-* Grant the service user `admin` rights in the repo
-* Since the `github.com/mozilla-iam` GitHub organization already whitelists the
-`AWS CodeBuild (Oregon)` third-party app, we don't need to temporarily whitelist
-the app in this step (unlike in, for example, the `github.com/mozilla` GitHub
-Organization)
-* In the AWS CodeBuild web console (in the appropriate AWS account), go to the
-  `Edit`... `Source` settings
-* Make changes and save
-* As a repo admin go back in [and remove the collaborator you added](https://github.com/mozilla-iam/auth0-custom-lock/settings/collaboration)
-* We don't need to un-whitelist the intergation at the GitHub Organization level
-  in the `github.com/mozilla-iam` GitHub org (as would be necessary in the
-  `github.com/mozilla` GitHub org)
-
-#### Deploying to Auth0-Dev manually
-
-For development:
-```bash
-./dmake deploy
-```
+To build for development or production, simply tag the repo with `development` or `production` tags, and GitHub
+should build and deploy. You can also build directly with `NODE_ENV=development npm run build` or
+`NODE_ENV=production npm run build`, although this should generally not be necessary.
 
-#### Deploying to Auth0-Prod automatically
+### Deploying code automatically
 
-Auth0-Prod is setup with an AWS CodeBuild job in `infosec-prod` (just like
-Auth0-Dev). To trigger a production deployment
+Both development and production code is deployed automatically via GitHub actions. Simply tag the `master` branch
```

Comment:
It isn't deployed via MAWS, as there is a service account created specifically for the S3 deployment. It's not easy to use MAWS here, since it builds inside a Docker container. It could be maybe be done, but I'm not sure it's worth the effort to figure out how given the long-term plan. I did add some notes otherwise.